projects/core/directives/dropdown/dropdown-context.directive.ts

```diff
@@ -44,6 +62,48 @@ export class TuiDropdownContextDirective extends TuiRectAccessor {
         this.driver.next(false);
     }
 
+    @HostListener('touchmove.silent', [
+        '$event.touches[0].clientX',
+        '$event.touches[0].clientY',
+    ])
+    onTouchMove(x: number, y: number): void {
+        if (!this.isIOS) {
+            return;
+        }
+
+        this.currentRect = tuiPointToClientRect(x, y);
+    }
+
+    @HostListener('touchstart.silent', [
+        '$event.touches[0].clientX',
+        '$event.touches[0].clientY',
+    ])
+    onTouchStart(x: number, y: number): void {
+        if (!this.isIOS) {
+            return;
+        }
+
+        this.currentRect = tuiPointToClientRect(x, y);
+        this.zone.runOutsideAngular(() => {
+            this.longTapTimeout = this.win.setTimeout(() => {
```

Comment:
No need for `win` here, this event will not happen in the context where `setTimeout` does not exist ever.

Reply:
https://github.com/taiga-family/taiga-ui/pull/6721#discussion_r1483218811

Reply:
or you can

```ts
setTimeout(() => ...) as unknown as number;
```

Reply:
Do you think is more beautiful than `any`?

Reply:
fixed

Reply:
You agree, I'm sorry, you can use `any` if you want

Reply:
fixed